### PR TITLE
[ #502 ] Fix URL given when uninstalling the operator and giving more info

### PR DESCRIPTION
**Pull request by gabriel-farache**
**Issue:** #502 

**Description:** The URL provided when uninstalling the operator was not correct. This PR fixes the URL and also add versioned URL when installing

Please make sure that your PR meets the following requirements:

- [X] You have read the [contributions doc](https://github.com/apache/incubator-kie-kogito-docs/blob/main/CONTRIBUTING.md)
- [X] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [X] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [X] The nav.adoc file has a link to this guide in the proper category
- [X] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 1]
```diff
@@ -142,17 +147,18 @@ kubectl get deployment sonataflow-operator-controller-manager -n sonataflow-oper
...
 ----
-kubectl delete -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/<version>.x/operator.yaml
+kubectl delete -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/v<version>/operator.yaml
 ----
 
 [TIP]
 ====
 If you're running a snapshot version, use this URL instead `https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/operator.yaml`.
+
+The URL should be the same used when installing the operator
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:161]:
```suggestion
The URL should be the same used when installing the operator.
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-= Install the {operator_name}
+https://xxx[= Install the ]{operator_name}
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:1]: Probably a leftover?
```suggestion
= Install the {operator_name}
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 3]
```diff
@@ -142,17 +147,18 @@ kubectl get deployment sonataflow-operator-controller-manager -n sonataflow-oper
 quay.io/kiegroup/kogito-serverless-operator-nightly:1.41.0
 ----
 
-The operator manager image reflects the current operator's version. Replace the major and minor version names in the command below. For example, if the image version is `1.41.0` use `1.41` in the placeholder:
-
 .Uninstalling the operator
+
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:151]:
```suggestion
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc (hunk 2 above)
Comment (MarianMacik) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:1]: @gabriel-farache is this a leftover?

Comment (gabriel-farache) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc]: yes, I wild lost copy/paste I think

**Pull request merged**